README.md

```diff
@@ -7,22 +7,26 @@
 This repository hosts source code implementing the Trusted Computing Group's (TCG) TPM2 Software Stack (TSS).
 This stack consists of the following layers from top to bottom:
 
-* Enhanded System API (ESAPI) as described in the  [TSS 2.0 Enhanced System API (ESAPI) Specification](https://trustedcomputinggroup.org/wp-content/uploads/TSS_ESAPI_Version-0.9_Revision-04_reviewEND030918.pdf).
+* Enhanded System API (ESAPI) as described in the [TSS 2.0 Enhanced System API (ESAPI) Specification](https://trustedcomputinggroup.org/wp-content/uploads/TSS_ESAPI_Version-0.9_Revision-04_reviewEND030918.pdf).
 This API is a 1-to-1 mapping of the TPM2 commands documented in Part 3 of the TPM2 specification.
 Additionally there are asynchronous versions of each command.
 In addition to SAPI, the ESAPI performs tracking of meta data for TPM object and automatic calculation of session based authorization and encryption values.
-Both the synchronous and asynchronous API are exposed through a single library: libesapi.
-* System API (SAPI) as described in the  [system level API and TPM command transmission interface specification](http://www.trustedcomputinggroup.org/resources/tss_system_level_api_and_tpm_command_transmission_interface_specification).
+Both the synchronous and asynchronous API are exposed through a single library: libtss2-esys.
+* System API (SAPI) as described in the [system level API and TPM command transmission interface specification](http://www.trustedcomputinggroup.org/resources/tss_system_level_api_and_tpm_command_transmission_interface_specification).
 This API is a 1-to-1 mapping of the TPM2 commands documented in Part 3 of the TPM2 specification.
 Additionally there are asynchronous versions of each command.
 These asynchronous variants may be useful for integration into event-driven programming environments.
-Both the synchronous and asynchronous API are exposed through a single library: libsapi.
+Both the synchronous and asynchronous API are exposed through a single library: libtss2-sys.
+* Marshaling/Unmarshaling (MU) as described in the [TCG TSS 2.0 Marshaling/Unmarshaling API Specification](https://trustedcomputinggroup.org/wp-content/uploads/TSS_Marshaling_Unmarshaling_API_Version-1.0_Revision-04_review_ENDS030918.pdf).
```

Comment:
s:Marshaling/Unmarshaling:Marshalling/Unmarshalling

Reply:
Both are correct "marshalling or marshaling" (see https://en.wikipedia.org/wiki/Marshalling_%28computer_science%29).

Since the API uses 1 L, the doc should do so too...

Reply:
OK--interesting.  Maybe it's a British/Americanism difference. It looks wrong to me, but it's best to be consistent with the API spec.

Reply:
skipped